cores/esp8266/Updater.cpp

```diff
@@ -236,7 +236,7 @@ bool UpdaterClass::end(bool evenIfRemaining){
     DEBUG_UPDATER.printf_P(PSTR("[Updater] Adjusted binsize: %d\n"), binSize);
 #endif
       // Calculate the MD5 and hash using proper size
-    uint8_t buff[128];
+    uint8_t buff[128] __attribute__((aligned(4)));
```

Comment:
This is 128 bytes on our stack of 4KB. There is a Esp.flashRead() that follows, which uses up an additional 512 bytes of stack.

Reply:
`Esp.flashRead()` will not use 512 bytes since `buff` is aligned, that's why I added `__attribute__((aligned(4)))`